opendebates/moderator_views.py

```diff
@@ -51,10 +65,62 @@ def mark_duplicate(request):
     if request.POST.get("send_email") == "yes":
         if request.POST.get("handling") == "merge":
             send_email("your_idea_is_merged", {"idea": to_remove})
-            send_email("idea_merged_into_yours", {"idea": to_remove})
+            send_email("idea_merged_into_yours", {"idea": duplicate_of})
         else:
             if duplicate_of is not None:
                 send_email("idea_is_duplicate", {"idea": to_remove})
             else:
                 send_email("idea_is_removed", {"idea": to_remove})
-    return redirect(".")
+    messages.info(request, msg)
+    return redirect('moderation_home')
+
+
+@rendered_with("opendebates/moderation/remove.html")
```

Comment:
This is called from the moderation_home page. For each flagged submission, the admin can choose to 'accept' or 'reject' the flag, which POSTs to this form: https://github.com/caktus/django-opendebates/blob/op-44-45-flags/opendebates/templates/opendebates/moderation/home.html#L33-L41